### DISCUSSION TITLE
Hy Language Server

### DISCUSSION THREAD

**Discussion by rajp152k** [hylang/hy#2535] [General]
Implementation of a language server somewhere down the lane?
I'd like to help if it's in the works..

**Comment by Kodiologist** [comment]
Not that I know of.

**Comment by sakurai-andrew** [comment]
I'm developing a language server for hylang(hy version 0.24 or later).
It's still in the experimental stage, and all Pull Requests are welcome.

https://github.com/sakuraiyuta/hyuga

**Reply by rajp152k** [reply]
Cool, will check it out and see how I could help

**Comment by atisharma** [comment]
Probably you meant LSP (hyuga) but if you wanted nREPL there was https://github.com/allison-casey/HyREPL

Edit: looks like https://github.com/masatoi/HyREPL is being maintained.
